src/features/views/components/ViewDataTable/ViewDataTableToolbar.tsx

```diff
@@ -55,16 +57,23 @@ const ViewDataTableToolbar: React.FunctionComponent<
   };
   return (
     <Box role="toolbar">
-      <Slide direction="left" in={!!selection.length} timeout={150}>
-        <Button
-          data-testid="ViewDataTableToolbar-createFromSelection"
-          disabled={disabled}
-          onClick={onViewCreate}
-          startIcon={<Launch />}
-        >
-          <Msg id={messageIds.toolbar.createFromSelection} />
+      {!isLoading && (
```

Comment:
Right now, you technically have two separate buttons for the loading vs not loading state. In my head, I think of them as the same button, but in two different states. That doesn't mean that they can't be implemented separately, but I see no reason to, so I would just remove this condition (and the entire other block) and just pass conditional properties to the single button based on `isLoading`.

Reply:
I totally agree with this solution and avoiding having 2 buttons, I'll do the changes asap :)